### PR TITLE
Update list of available clickhouse table functions

### PR DESCRIPTION
**Pull request by Khatskevich**
Update list of available clickhouse table functions.

### CODE REVIEW

File: docs/products/clickhouse/reference/supported-table-functions.rst [hunk 1]
```diff
@@ -10,13 +10,17 @@ Table functions supported in Aiven for ClickHouse®
       SELECT *
       FROM deltaLake('s3://bucket/path/to/lake')
 
+Some of the table functions are explicitly disabled due to security concerns.
```
Comment (wojcik-dorota) [inline docs/products/clickhouse/reference/supported-table-functions.rst:13]:
```
.. note::

   Occasionally, you may find specific table functions disabled for security reasons.
```

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

File: docs/products/clickhouse/reference/supported-table-functions.rst (hunk 1 above)
Comment (Khatskevich) [reply docs/products/clickhouse/reference/supported-table-functions.rst]: Frankly speaking I am not sure if this one reflects the Idea better.
But I am not a wording expert, I will trust you here)

**Comment by wojcik-dorota** [comment]
Looks good from stylistic and editorial pov. @aris-aiven @joelynch could you please approve from tech pov? Thank you :-)

File: docs/products/clickhouse/reference/supported-table-functions.rst [hunk 2]
```diff
@@ -10,13 +10,19 @@ Table functions supported in Aiven for ClickHouse®
...
+
+   You may find some specific table functions disabled for security reasons.
+
 Aiven for ClickHouse® supports the following table functions:
 
 * ``cluster``
 * ``clusterAllReplicas``
 * ``cosn``
 * ``deltaLake``
+* ``dictionary``
```
Comment (joelynch) [inline docs/products/clickhouse/reference/supported-table-functions.rst:23]: We don't allow dictionaries so this is a bit moot

**Review by joelynch — APPROVED** [review]
Looks good to me :)

File: docs/products/clickhouse/reference/supported-table-functions.rst (hunk 2 above)
Comment (Khatskevich) [reply docs/products/clickhouse/reference/supported-table-functions.rst]: you are right. mentioning it there will not help anyone. Deleted

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**